carball/analysis/stats/possession/ball_distances.py

```diff
@@ -31,13 +31,19 @@ def calculate_player_stat(self, player_stat_map: Dict[str, PlayerStats], game: G
             for players_data_frame in [closest_players, furthest_players]
         ], axis=1)
 
+        player_ball_distance_times.fillna(value=0, inplace=True)
```

Comment:
Does this not mess the stats up?

Reply:
no because it just sets the value to zero meaning you were never one of the values